recipes/telegram-send/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "telegram-send" %}
+{% set version = "0.25" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: e6cda001a89e1cd02317fcd0456288a0cdd5aa56970dec6fbd83ad9af1db05ac
+
+build:
+  number: 0
+  entry_points:
+    - telegram-send=telegram_send:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [py<=34]
+
+requirements:
+  host:
+    - appdirs
+    - colorama
+    - pip
+    - python
```

Comment:
```suggestion
    - python >=3.6
```

Reply:
I think you just need ``pip`` and ``python`` on ``host``. Could you please dele the other deps?